resource/src/routes/route.mustache

```diff
@@ -0,0 +1,57 @@
+const { Router } = require('express');
+const {{camelCasePlural}} = require('../controllers/{{snakeCasePlural}}');
+
+const serialize = ({{camelCase}}) => {
+  const data = Object.assign({}, {{camelCase}});
+  {{#isUser}}
+  delete(data.password);
+  {{/isUser}}
+  return data;
+};
+
+const router = module.exports = new Router()
+  .get('', function * (req, res) {
+    const list = yield {{camelCasePlural}}.all(req.query);
+    res.json(list.map(serialize));
+  })
+  .get('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.find(req.params.id);
+    res.json(serialize({{camelCase}}));
+  })
+  .post('', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.create(req.body);
+    res.status(201).json(serialize({{camelCase}}));
+  })
+  .put('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.update(req.params.id, req.body);
```

Comment:
yeah, i get the difference between the two. although i find PATCH is by far more useful on the front end than PUT. especially when you start to build those atomic change rich UIs. that why i thought you went this way with the original implementation.

we can have both, it's a minimal change anyways